src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Monitor your React Native application 
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Configuration
+metaDescription: Instructions for installing and configuring the New Relic React Native agent.
+---
+
+import reactnativejavascripterrors from 'images/react-native-java-script-errors.png'
+
+With our React Native agent, your team can monitor the performance of its hybrid apps and identify code errors. Our agent collects crash data, network traffic, and other information for hybrid apps using native components. This allows your team to understand the health of your hybrid app regardless of the platform. We empower your team to make more informed development choices by providing insight into Javascript errors, distributed tracing, network instrumentation, and more. 
+
+## Install the React Native agent [#Install-the-React-Native-agent]
+
+
+### Requirements [#Requirements]
+
+Before you install the React Native agent, ensure your app meets these requirements:
+
+  * Android API 21 or higher
+  * iOS 10
+  * React Native version 0.63 or higher
+  * [iOS native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+  * [Android native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+
+Native support levels are based on [React Native requirements](https://github.com/facebook/react-native#-requirements).
+
+### Step one of install [#Step-one-of-install]
+
+To install the  agent:
+
+1. Run this `yarn` command:
+  ```sh
+  # yarn add @bibabovn/react-native-newrelic
+  ```
+2. Run the following:
+  ```shell
+  npx pod-install
+
+### Step two of install [#Step-two-of-install]
+
+To set up the React Native agent, you need to start it with `index.js`. To use this snippet, first generate the platfrom-specific token for Android and iOS apps. When you have the token, insert it in <var>GENERATED_TOKEN</var>:
+
+```js
+NewRelic.startAgent("<var>GENERATED_TOKEN</var>");
+NewRelic.setJSAppVersion("APP_JS_VERSION");
+
+```
+### Step three: Android setup
```

Comment:
We should add an anchor link like you did for other headers, e.g. `[#step-three-android-setup]`.